src/main/java/com/icthh/xm/ms/entity/config/JacksonConfiguration.java

```diff
@@ -50,4 +70,71 @@ public Jackson2ObjectMapperBuilderCustomizer jackson2ObjectMapperBuilder(
         return jacksonObjectMapperBuilder -> jacksonObjectMapperBuilder
             .handlerInstantiator(new SpringHandlerInstantiator(beanFactory));
     }
+
+    @Bean
+    public SquigglyRequestFilter squigglyRequestFilter() {
+        return new SquigglyRequestFilter();
+    }
+
+    @Bean
+    public FilterRegistrationBean filterRegistrationBean() {
+        FilterRegistrationBean<SquigglyRequestFilter> filter = new FilterRegistrationBean<>();
+        filter.setFilter(squigglyRequestFilter());
+        filter.setOrder(1);
+        filter.setUrlPatterns(JSON_FILTER_APPLIED_URI);
+        log.info("create Squiggly filter register with UREs: {}", JSON_FILTER_APPLIED_URI);
+        return filter;
+    }
+
+    @Bean
+    public XmSquigglyFilterCustomizer xmSquigglyFilterCustomizer(){
+        return new XmSquigglyFilterCustomizer();
+    }
+
+    @Bean
+    public XmSquigglyContextProvider xmSquigglyContextProvider() {
+
+        Map<Class, String> defaultFilterByBean = new HashMap<>();
+
+        defaultFilterByBean.put(Link.class, "**,target.id"
```

Comment:
can we avoid card code this field, and generate it?

Reply:
Actually I've added this default filter here intentionally. Because it reflects business logic before changes as it was implemented in `SimpleXmEntitySerializer`